index.js

```diff
@@ -1,6 +1,8 @@
 const server = require('./server.js');
 require("dotenv/config")
```

Comment:
By having this here and not in the `server.js` it means that the `.env` file will not be loaded up for the test.
This will cause an error when attempting to access `process.env.SECRET` as a require parameter of `jwt.sign()`.

It would be better to either move this to `server.js` so it get's executed during the testing OR add some sort of documentation (like a JSDoc on the `server.js` export) so it is clear that this, and any other, environment variables need to be configured prior to starting the server.